app/views/RoomInfoView/index.js

```diff
@@ -21,10 +23,14 @@ import { withTheme } from '../../theme';
 import { themedHeader } from '../../utils/navigation';
 
 const PERMISSION_EDIT_ROOM = 'edit-room';
-
 const camelize = str => str.replace(/^(.)/, (match, chr) => chr.toUpperCase());
-const getRoomTitle = (room, type, name, theme) => (type === 'd'
-	? <Text testID='room-info-view-name' style={[styles.roomTitle, { color: themes[theme].titleText }]}>{name}</Text>
+const getRoomTitle = (room, type, name, username, theme) => (type === 'd'
+	? (
+		<>
+			<Text testID='room-info-view-name' style={[styles.roomTitle, { color: themes[theme].titleText }]}>{ name }</Text>
+			<Text testID='room-info-view-name' style={[styles.roomUsername, { color: themes[theme].auxiliaryText }]}>{ username && `@${ username }` }</Text>
```

Comment:
Change `testID` to `room-info-view-username` and make the condition before render the `<Text>`.
Like this:  `{ username ? <Text>{username}</Text> : null }`.

Reply:
Done